fastlane/metadata/android/es-ES/short_description.txt

```diff
@@ -0,0 +1 @@
+Relájate, mejora el enfoque y aumenta la productividad con un mínimo ruido de fondo.
```

Comment:
We need to keep it under 80 characters. Its a restriction from the Play Store.

Reply:
Done, also I adjusted two translation values. :wink: